timers.c

```diff
@@ -565,7 +565,7 @@
     }
 /*-----------------------------------------------------------*/
 
-    static portTASK_FUNCTION( prvTimerTask, pvParameters )
+    static portTASK_FUNCTION( prvTimerTask, pvParameters ) portNORETURN;
```

Comment:
This does not seem correct?

Reply:
What is `this` ? What is not correct about it?